agent/ansible/collection/roles/pbench_agent_install/tasks/main.yml

```diff
@@ -5,7 +5,7 @@
 
 - name: Install RPMs
   ansible.builtin.package:
-    name: "{{ item }}"
-  with_items:
-    - pbench-agent
-    - pbench-sysstat
+    name:
+      - pbench-agent
+      - pbench-sysstat
+    state: latest
```

Comment:
Springboarding from [Dave's comment](https://github.com/distributed-system-analysis/pbench/pull/3478#pullrequestreview-1514296991), is `latest` really the appropriate token here?  Should we not instead refer to something which is specific to the v0.72.1 release or at least to the output of the `b0.72` branch?

Or, perhaps, Dave and I are thinking about this wrong:  _this_ commits revisions of these files is the "gold" one which would/should be uploaded to Galaxy, and the one on `main` is the basis for the contents of the next release and which will then be pushed to replace this one -- we don't have two copies -- we have the live, "released" one, and we have the one on `main`.

Reply:
> Or, perhaps, Dave and I are thinking about this wrong: _this_ commits revisions of these files is the "gold" one which would/should be uploaded to Galaxy, and the one on `main` is the basis for the contents of the next release and which will then be pushed to replace this one -- we don't have two copies -- we have the live, "released" one, and we have the one on `main`.

OK; yeah, one way to think about it is that both branches currently have identical Ansible code, and we'd update the version on `main` only when/if we change it. On the other hand, unless we change the code there's no reason to push a Galaxy package from `main`, and I'd prefer that we avoid the risk of accidentally failing to update the version by doing it proactively. 🦞

Reply:
Yup, it's the latter. If we need changes in the next release, we'll build and upload a new collection  from `main` (and bump the version number: Galaxy refuses the upload if the version number exists already). When testing for a release, I use the collection in `main` (or the release branch created from it), not the stuff that is in Galaxy.